src/main/kotlin/edu/ycp/cs482/iorcapi/factories/DetailFactory.kt

```diff
@@ -102,6 +107,11 @@ class DetailFactory(
         return ClassQL(rpgClass)
     }
 
+    fun deleteClass(id:String):String{
+        classRepository.delete(id)
```

Comment:
What does this do if the id doesn't exist? Probably nothing. Should look into that

Reply:
Nothing on either the server or graphql outputs